src/sonic-frr/patch/0033-bgpd-backpressure-cleanup-bgp_zebra_XX-func-args.patch

```diff
@@ -105,11 +113,10 @@ index 69240a3b83..920df835a4 100644
 -				     safi, true);
 +	if (table->safi == SAFI_FLOWSPEC) {
 +		bgp_pbr_update_entry(bgp, p, info, table->afi, table->safi,
-+				     true);
++							true);
```

Comment:
why the whitespace change?

Reply:
The patch is auto generated from the commit by "git format-patch -1" command.